x-pack/plugins/cloud_security_posture/public/components/cloud_security_data_table/additional_controls.tsx

```diff
@@ -39,6 +40,18 @@ export const AdditionalControls = ({
   const closeModal = () => setIsFieldSelectorModalVisible(false);
   const showModal = () => setIsFieldSelectorModalVisible(true);
 
+  const GroupSelector = () => {
```

Comment:
does it makes sense to move it outside of the component as `GrouptSelectorWrapper` or smth?

Reply:
yes nice suggestion